src/gausskernel/storage/access/k2/pg_gate_api.cpp

```diff
@@ -200,14 +200,19 @@ K2PgStatus PgGate_ExecCreateDatabase(const char *database_name,
                                  K2PgOid next_oid) {
   elog(LOG, "PgGateAPI: PgGate_ExecCreateDatabase %s, %d, %d, %d",
          database_name, database_oid, source_database_oid, next_oid);
-  return K2PgStatus::NotSupported;
+  return pg_gate->GetCatalogClient()->CreateDatabase(database_name,
+      k2pg::PgObjectId::GetDatabaseUuid(database_oid),
```

Comment:
please double check if we use the database_oid as a string directly or uuid of it in the original chogori-sql logic. I kinda of remember that we mapped database_oid to k2 collection directly, not database's uuid.

Reply:
Database id (uuid) seems be used to create collection https://github.com/futurewei-cloud/chogori-sql/blob/master/src/k2/connector/pggate/catalog/sql_catalog_manager.cc#L342. Note that both database uuid (database id) and oid is passed to the function. The function internally decides which one to use.